apps/checkout-app/__tests__/backend/middlewares.test.ts

```diff
@@ -0,0 +1,57 @@
+import { withSaleorDomainMatch } from "@/checkout-app/backend/middlewares";
+import { getSaleorDomain } from "@/checkout-app/backend/utils";
+
+jest.mock("@/checkout-app/backend/utils.ts");
+
+const mockedGetSaleorDomain = <jest.Mock>getSaleorDomain;
```

Comment:
I'm not sure, but wouldn't this work?

`const mockedGetSaleorDomain = jest.mock("@/checkout-app/backend/utils.ts");`